libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/bottomsheet/CustomBottomSheetScaffold.kt

```diff
@@ -253,9 +253,9 @@ private fun CustomStandardBottomSheet(
             .onSizeChanged { layoutSize ->
                 val newAnchors = calculateAnchors(layoutSize)
                 val newTarget = when (state.anchoredDraggableState.targetValue) {
-                    SheetValue.Hidden, SheetValue.PartiallyExpanded -> SheetValue.PartiallyExpanded
-                    SheetValue.Expanded -> {
-                        if (newAnchors.hasAnchorFor(SheetValue.Expanded)) SheetValue.Expanded else SheetValue.PartiallyExpanded
+                    SheetValue.Hidden, PartiallyExpanded -> PartiallyExpanded
```

Comment:
`SheetValue.Hidden` has not been touch like the others, maybe add the import like for the others?

Reply:
Ah sorry, moving the preview annotations added fully qualified imports everywhere and it seems like I inlined some imports that I didn't mean to while trying to fix this in a single operation.